obtener y mostrar salario mensual

Juni-p opened the pull request.

Juni-p committed:
obtener y mostrar salario mensual

hernan-peralta commented on tarea-1/index.js:
```diff
@@ -0,0 +1,11 @@
+);
+
+$btnIngresoMensual.onclick = function () {
+  const MESES = 12;
```
En mi opinión el nombre de esta variable no refleja su verdadero proposito, que es guardar la cantidad de meses en un año. Podría ser algo como CANTIDAD_MESES, MESES_EN_UN_ANIO o algo asi.

Juni-p replied:
Perfecto hernan, voy a corregir eso!

Juni-p committed:
cambia la variable meses

modifico el nombre de la variable para tener mayor claridad a la hora de leer el codigo
